nixos/modules/services/torrent/flood.nix

```diff
@@ -30,6 +30,13 @@ in
       default = "localhost";
       example = "::";
     };
+    runDir = lib.mkOption {
```

Comment:
Could this be renamed to something like `stateDir` to avoid confusion with `XDG_RUNTIME_DIR`/`RuntimeDirectory` meant for ephemeral items?

Edit: just noticed that the main executable uses the `--rundir` flag. Feel free to ignore this comment if you think more people using the module will be familiar with flood's cli flags.